### DISCUSSION TITLE
Turborepo deploy to aws

### DISCUSSION THREAD

**Discussion by McCo0L** [vercel/turborepo#1418] [Help] [answered]
I'm moving a Serverless Nextjs app to Turborepo and need it deployed on AWS, but cannot find any example or clue on how to deploy Turborepo to AWS. 
Does someone has an example or clue of what I should be using to deploy it correctly? Like managing CDN, Static and dynamic parts of the application and get them deployed to AWS.

**Comment by Rafik-Belkadi-Reccap** [comment]
have you found  something ?

**Reply by McCo0L** [reply]
nope, nothing.  Looks like they only want this stuff running on Vercel.

**Reply by nathanhammond** [reply]
Turborepo is a build system; we run your build. You can deploy built artifacts to any service that you want. Whatever your deploy setup is before adopting Turborepo is 100% compatible with Turborepo.

If you don't have a deployment setup we're of course going to recommend that you use Vercel's. Vercel will run your build for you and then create a deploy from the assets created during that build.

**Comment by nathanhammond** [comment, marked as answer]
Turborepo is a build system; we run your build. You can deploy built artifacts to any service that you want. Whatever your deploy setup is before adopting Turborepo is 100% compatible with Turborepo.

If you don't have a deployment setup we're of course going to recommend that you use Vercel's. Vercel will run your build for you and then create a deploy from the assets created during that build.
